pwa/src/templates/templateParts/submitComponent/SubmitComponentTemplate.tsx

```diff
@@ -30,6 +30,6 @@ export const SubmitComponentTemplate: React.FC<SubmitComponentTemplateProps> = (
           Component toevoegen
         </Button>
```

Comment:
The button does not seem to have an icon anymore.
And the button uses the wrong styling.